Test/DatabaseTests/TestProteinReader.cs

```diff
@@ -388,6 +390,10 @@ public static void TestSlideDecoyXML()
                 Assert.AreEqual(ok2[1].BaseSequence[bond.OneBasedBeginPosition - 1], 'C');
                 Assert.AreEqual(ok2[1].BaseSequence[bond.OneBasedEndPosition - 1], 'C');
             }
+
```

Comment:
Can you expand on this test to make sure that results of decoy generation are as expected, not just that it does not loop indefinitely.

Reply:
Good call. Caught some bugs.